index.html

```diff
@@ -1,28 +1,34 @@
 <!DOCTYPE html>
 <html>
-<head>
-  <title>My Portfolio</title>
-  <link rel="stylesheet" href="style.css">
-</head>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>My Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+  </head>
 <body>
-  <nav class="navbar">
-    <h1>MY Logo</h1>
-    <ul class="nav-menu">
-      <li><a href="#">Home</a></li>
-      <li><a href="#">About</a></li>
-      <li><a href="#">Contact</a></li>
-    </ul>
-    <img  class="hamburger" src="assets/menu.png" width="50px"  alt="Logo" />
+  <header class="header">
+    <nav class="navbar">
+      <h1 ><a class="my-logo" href="#section2">MY Logo </a></h1>
+      <ul class="nav-menu">
+        <li><a href="#">Portfolio</a></li>
+        <li><a href="#">About</a></li>
+        <li><a href="#">Contact</a></li>
+      </ul>
+      <img  class="hamburger" src="assets/menu.png" width="50px"  alt="Logo" />
     </nav>
-    <section class="section2-content">
-      <h1>I’m Melkamu Microverse student, Glad to see you!</h1>
-      <p>I’m a software developer! I can help you build a product feature or website Look through some of my 
+  </header>
+  <section id="section2" class="section2-content">
+      <p class="my-title"> I’m Melkamu,<br> Glad to see you!</p>
+      <p class="my-description">I’m a software developer! I can help you build a product feature or website Look through some of my 
           work and experience! If you like what you see and have a project you need coded don’t hesitate to contact me.</p>
-      <a class="contact-label" href="#">LET’S CONNECT</a> 
-      <a class="my-address-links"href="#"><img src="./assets/m.png"></a>
-      <a class="my-address-links" href="#"><img src="./assets/git.png"></a>
-      <a class="my-address-links" href="#"><img src="./assets/t.png"></a>
-      <a class="my-address-links" href="#"><img src="./assets/lin.png"></a>
+      <p class="contact-label">LET’S CONNECT</p> 
+      <ul>
+      <li class="my-address-list"> <a class="my-address-links" href="https://github.com/melkamu372"><img src="./assets/git.png"></a> </li> 
+      <li class="my-address-list">  <a class="my-address-links" href="https://www.linkedin.com/in/melkamu372"><img src="./assets/lin.png"></a> </li>
+      <li class="my-address-list"><a class="my-address-links"href="#"><img src="./assets/m.png"></a> </li> 
+      <li class="my-address-list"> <a class="my-address-links" href="#"><img src="./assets/t.png"></a> </li>   
+    </ul>
```

Comment:
- [ ] [2] I recommend referring to the Figma design for the correct spacing of the social media icons. Currently, they appear to be too spaced out, so adjusting them according to the design will help maintain consistency and improve the overall aesthetics. 😄